.github/workflows/build-mbed-cli-1.yaml

```diff
@@ -0,0 +1,30 @@
+name: Mbed CLI 1
+on:
+  - push
+  - workflow_dispatch
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
It seems to me that `fetch-depth` = 0 is to retrieve the whole repo history (see https://github.com/actions/checkout#usage). I'm not sure this is required here.

Reply:
Yah, I normally add it to new projects or those with small histories as a default. Mostly because when versioning with git tags and using `git describe` then it is hard to find the right depth that corresponds to the number of commits since the last tag.

@mphalke I assume since the repository is fairly new it doesn't have a version yet? Do you want to version the repository? Looks like no-OS isn't versioned, HDL is `2021_R2` as a annotated tag, libiio is `v0.25` as a annotated tag.